src/components/SkillPage/SkillListing.js

```diff
@@ -149,6 +154,18 @@ class SkillListing extends Component {
                     self.saveSkillUsage()
                 }
             });
+            // Fetch country wise skill usage of the visited skill
+            $.ajax({
+                url: countryWiseSkillUsageUrl,
+                dataType: 'json',
+                crossDomain: true,
+                success: function (data) {
+                    self.saveCountryWiseSkillUsage(data.skill_usage)
+                },
+                error: function(e) {
+                    self.saveCountryWiseSkillUsage()
```

Comment:
Update this later.

Reply:
Yes that can work too, but it's usually advisable to log the error for easy debugging purpose. :+1: